### DISCUSSION TITLE
Add a preview to the Transmog Npc, also allow any armor type for xmog.

### DISCUSSION THREAD

**Discussion by MichaelCrow** [chromiecraft/chromiecraft#4924] [Suggestions]
Id like to ask for a preview function or for the npc to whisper you a link of the item you want to xmog before doing so that you may preview/check out the display before paying for it.

Additionally, leveling as a warrior gave me access to a lot of cool low level mail xmogs, that I can no longer use because I wear plate, it would be sick to allow cross armor type xmogs or to allow xmog if you have the required proficiencies for your class to all known armor types.

**Comment by heyitsbench** [comment]
I would recommend the [transmog addons](https://github.com/ZhengPeiRu21/transmog-addons) pinned in the CC #addons channel.

**Reply by MichaelCrow** [reply]
Cool, what about adding the ability to make plate look like mail?

Also what about armor that doesnt have a type like the Lovely Black Dress or other seasonal items?

**Reply by heyitsbench** [reply]
Already suggested [here](https://github.com/chromiecraft/chromiecraft/discussions/3299).
